Dockerfile

```diff
@@ -2,18 +2,20 @@ FROM oven/bun AS builder
 
 WORKDIR /app
 
+ARG PUB_ENV
 ARG PUB_HOSTNAME
 ARG PUB_PLAUSIBLE_URL
 
+ENV PUB_ENV=${PUB_ENV}
```

Comment:
a default value for this is missing on the `docker-compose.yml` file

Reply:
added